packages/analytics-js-plugins/src/errorReporting/event/event.ts

```diff
@@ -73,14 +73,10 @@ const normaliseError = (
   let error;
```

Comment:
We don't need any complex logic to handle different types of the errors. Also, remove tolerateNonErrors input.

if (error) {
  // proceed
} else {
  // log warning
  // proceed
}

// throw error if needed.